scripts.js

```diff
@@ -1,32 +1,92 @@
 // FILL IN THE FUNCTIONS BELOW
 
 var sprintFunctions = {
-  largestEl: function(){  
+  largestEl: function(array){
     // your code here
+    return Math.max.apply(Math, array);
   },
-  
-  reversed: function(){  
+
+  reversed: function(string){
     // your code here
+    return string.split("").reverse().join("");
   },
 
-  loudSnakeCase: function(){  
+  loudSnakeCase: function(sentence){
     // your code here
+    var result = [];
+    var sentence_array = sentence.split(" ");
+    var alphanumeric;
+    sentence_array.forEach(function(element){
+      alphanumeric = element.replace(/[^a-z0-9]/gi,'');
+      if(alphanumeric !== ""){
+      result.push(alphanumeric[0].toUpperCase() + alphanumeric.slice(1));
+      }
+    });
+    return result.join("_");
   },
 
-  compareArrays: function(){ 
+  compareArrays: function(array_1, array_2){
     // your code here (replace the return)
-    return "Finish compareArrays first!" 
+    if (array_1.length !== array_2.length){
+      return false;
+    }
+    var false_flag = true;
+    array_1.forEach(function(element, index){
+      console.log(element + " " + array_2[index]);
+      if (element != array_2[index]){
+        // return false; WTF doesn't work?!? Note to self: SO this
```

Comment:
It wasn't breaking out after running line 37 (return false) when I checked it line-by-line using Chrome dev tools. I ended up setting the false_flag variable and returning it instead.